distribution/docs/src/main/resources/_contents/_core-concepts/registry-intro-contents.adoc

```diff
@@ -0,0 +1,4 @@
+
+== Registry
+
+${ddf-registry} is a
```

Comment:
== Registry

The Registry Application serves as an index of registry nodes and their information, including service bindings, configurations and supplemental details.

Each registry has the capability to serve as an index of information about a network of registries which, in turn, can be used to connect across a network of DDFs. Registries communicate with each other through the CSW endpoint and each registry node is converted into a registry metacard to be stored in the catalog. When a registry is subscribed to or published from, it sends one or more nodes to another registry. 

=== Identity Node

The DDF Registry is initially comprised of a single registry node, refered to as the identity, which represents the registry's primary configuration.

=== Subscription

Subscribing to a registry is the act of retreiving its information, specifically its identity information and any other registries it knows about. By default subscriptions are configured to check for updates every 30 seconds.

=== Publication

Publishing is the act of sending a registry's information to another registry. Once publication has occurred, any updates to the local registry will be pushed out to the registries that have been published to.